core_impl/src/main/java/io/opencensus/implcore/stats/MutableAggregation.java

```diff
@@ -299,6 +299,18 @@ void combine(MutableAggregation other, double fraction) {
       checkArgument(this.bucketBoundaries.equals(mutableDistribution.bucketBoundaries),
           "Bucket boundaries should match.");
 
+      // Algorithm for calculating the combination of sum of squared deviations:
+      // https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Parallel_algorithm.
+      double delta = mutableDistribution.mean - this.mean;
+      double ma = this.sumOfSquaredDeviations * (this.count - 1);
+      double mb = mutableDistribution.sumOfSquaredDeviations * (mutableDistribution.count - 1);
```

Comment:
I think these should just be sumOfSquaredDeviations--the pseudocode from Wikipedia takes variances as arguments (using the unbiased estimator formula) and thus needs to multiply by N-1 to get back the sum of the squared deviations, but you already have the sum rather than the variance.

The test looks correct, but both the distributions have count == 2, so multiplying by count - 1 does nothing--I think it will fail if you give it a 3+ element distribution.

Reply:
You're right, fixed. Also added a test for the case when there are >2 values.